web/src/components/Upstream/UpstreamForm.tsx

```diff
@@ -133,9 +133,9 @@ const UpstreamForm: React.FC<Props> = forwardRef(
               <Form.Item
                 required
                 key={field.key}
-                label={index === 0 && '节点域名/IP'}
+                label={index === 0 && formatMessage({ id: 'upstream.step.node.domain.or.ip' })}
```

Comment:
I recommend to name the key refer to:  [basicModule].[moduleName].[elementName].[...desc]

```suggestion
                label={index === 0 && formatMessage({ id: 'page.upstream.form.itemLabel.nodeDomainOrIp' })}
```